packages/ubuntu_provision/lib/src/identity/identity_model.dart

```diff
@@ -134,7 +134,8 @@ class IdentityModel extends SafeChangeNotifier with PropertyStreamNotifier {
 
   /// Whether the current input is valid.
   bool get isValid {
-    return realName.length <= kMaxRealNameLength &&
+    return realName.isNotEmpty &&
```

Comment:
Does subiquity not allow an empty real name? Because `/etc/passwd` does allow empty real names.

Reply:
Ah, I just read the issue, so it gets some problem on the way, I guess it's fine then!
The test checking that the real name should be allowed to be empty has to be updated though.

Reply:
In Subiquity it seems like it is handling empty realname by copying username to realname when it's empty:

https://github.com/canonical/subiquity/blob/main/subiquity/models/identity.py